appium/tests/screenobjects/sent.screen.ts

```diff
@@ -0,0 +1,43 @@
+import BaseScreen from './base.screen';
+import TouchHelper from "../helpers/TouchHelper";
+
+const SELECTORS = {
+  SENT_HEADER: '-ios class chain:**/XCUIElementTypeNavigationBar[`name == "SENT"`]',
+  SEARCH_ICON: '~search icn',
+  HELP_ICON: '~help icn'
+};
+
+
+class SentScreen extends BaseScreen {
```

Comment:
Inbox, sent, trash, and other such screens are all the same - they are all just a folder screen. So it should probably really just be one screen. A `MailFolderScreen` or maybe just `InboxScreen` or if you want to extend from something and still have them all separate, then `BaseFolderScreen`. What do you think?